### PR TITLE
Use gradient boosting instead of ridge

### PR DESCRIPTION
**Pull request by GaelVaroquaux**
Tree based models work better. Trying this out to see if the example takes too long.

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
This is now ready for review. The usage pattern is now a bit more realistic.

File: examples/02_fit_predict_plot_employee_salaries.py [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
-We choose to benchmark different categorical encodings for
-the dirty column *Employee Position Title*, that contains
-dirty categorical data.
+Comparing encoders of a dirty categorical columns
+==================================================
 
+Considering a database on `employee salaries
+<https://catalog.data.gov/dataset/employee-salaries-2016>`_, one problem
+is that the *Employee Position Title* column contains dirty categorical
+data.
```
Comment (LilianBoulard) [inline examples/02_fit_predict_plot_employee_salaries.py:8]:
```suggestion
The column *Employee Position Title* of the dataset `employee salaries
<https://catalog.data.gov/dataset/employee-salaries-2016>`_ contains dirty categorical
data.
```

That's how I would rephrase it.
Also, the link is dead.

**Review by LilianBoulard — APPROVED** [review]

File: examples/02_fit_predict_plot_employee_salaries.py [hunk 2]
```diff
@@ -151,5 +140,14 @@ def make_pipeline(encoding_method):
 plt.yticks(size=20)
 plt.tight_layout()
 
+##########################################################################
+# The clear trend is that encoders that encoders that use the string form
```
Comment (LilianBoulard) [inline examples/02_fit_predict_plot_employee_salaries.py:143]:
```suggestion
# The clear trend is that encoders that use the string form
```

**Review by LilianBoulard — CHANGES_REQUESTED** [review]

**Review by LilianBoulard — APPROVED** [review]
LGTM !
